runtime/compiler/control/HookedByTheJit.cpp

```diff
@@ -475,6 +514,20 @@ static void jitHookInitializeSendTarget(J9HookInterface * * hook, UDATA eventNum
                      TR::Options::getHighCodeCacheOccupancyBCount() :
                      TR::Options::getHighCodeCacheOccupancyCount();
          }
+#if defined(J9VM_OPT_JITSERVER)
+      else if (methodCachedAtServer) // Higher priority than SCC
+         {
+         int32_t scount = optionsAOT->getInitialSCount();
```

Comment:
Please check: if local shared classes is not used, `optionsAOT` may not exist.

Reply:
You may use `countInOptionSet || TR::Options::getCountsAreProvidedByUser()` to check whether the use has specified counts.